app/controllers/collection_drafts_controller.rb

```diff
@@ -282,12 +282,68 @@ def validate_parameter_ranges(errors, metadata)
   end
 
   def validate_paired_fields(errors, metadata)
+    errors = validate_additional_attribute_value_field(errors, metadata)
     errors = validate_parameter_ranges(errors, metadata)
     errors = validate_project_paired_dates(errors, metadata)
     errors = validate_temporal_paired_dates(errors, metadata)
     validate_tiling_identification_systems_paired_fields(errors, metadata)
   end
 
+  # The following regex are used to validate AdditionalAttributes/#{index}/DataType, as
```

Comment:
Thomas helped me dig up the validation that CMR uses and I created regex for it. CMR uses Java parsing functions that accepted the formats outlined below. (It would've been nice if there was regex I could copy but the regex I created works)

Reply:
I think it is worthwhile/helpful to a future dev to add a small note about CMR using parsing functions and the regex is an approximation of their checks

Reply:
Will do.